tasks/3_is-boolean/is-boolean.js

```diff
@@ -0,0 +1,9 @@
+function isBoolean(bool) {
+  if (typeof(bool) === 'boolean') {
```

Comment:
`typeof` is an operator. Parentheses is not required here. Moreover if it is simple condition - probably you can just return it